Index.html

```diff
@@ -9,6 +9,8 @@
     <script src='main.js'></script>
 </head>
 <body>
+    <p>Hello This is Code jail</p>
+    <p>hello</p>
```

Comment:
try to add more code heading as well before a paragraph @robby3689 

- [ add heading tag ]